accepted/PSR-16-simple-cache.md

```diff
@@ -70,8 +70,8 @@ by an integer representing time in seconds, or a DateInterval object.
 cached item. Implementing libraries MUST support keys consisting of the
 characters `A-Z`, `a-z`, `0-9`, `_`, and `.` in any order in UTF-8 encoding and a
 length of up to 64 characters. Implementing libraries MAY support additional
-characters and encodings or longer lengths, but must support at least that
-minimum.  Libraries are responsible for their own escaping of key strings
+characters and encodings or longer lengths, but MUST support at least that
+minimum. Libraries are responsible for their own escaping of key strings
```

Comment:
This one is tricky; changing `must` to `MUST` changes the meaning from an informal requirement to a spec-formal requirement.  On the other hand, it's very clear from context (and from PSR-6, for which I was the editor, which shares this text) that `MUST` is the intent here.

So... I'm not sure.  I'm probably OK here?

Reply:
That's exactly what I thought while reading it. I read the sentence then I thought "Hey this is part of the recommendation, right?". Also, it's following another part of the sentence that's strongly linked to it and that uses a capitalized version as a start. Again, I proposed this for cohesion only.

Reply:
I agree, this seemed like the original intent. I'm :+1: on this, any library doing otherwise is breaking interop.